hawc/apps/study/templates/study/study_detail.html

```diff
@@ -39,13 +39,19 @@ <h2 class="d-inline-block">{{object}}</h2>
               <a class="dropdown-item" href="{% url 'invitro:celltype_create' object.pk %}">Create new cell-type</a>
             {% endif %}
 
-            {% if object.epi %}
+            {% if object.epi and object.assessment.epi_version == 1 %}
               <div class="dropdown-divider"></div>
               <span class="dropdown-header">Epidemiology editing</span>
               <a class="dropdown-item" href="{% url 'epi:sp_create' object.pk %}">Create new study population</a>
               <a class="dropdown-item" href="{% url 'epi:sp_copy_selector' object.pk %}">Copy from existing</a>
             {% endif %}
 
+            {% if object.epi and object.assessment.epi_version == 2 %}
+              <div class="dropdown-divider"></div>
+              <span class="dropdown-header">Epidemiology editing</span>
+              <a class="dropdown-item" href="{% url 'epiv2:des_create' object.pk %}">Create new study population</a>
```

Comment:
i would change this slightly. i'd put the epi_version if statement nested in the object.epi, and just switch the a links instead of the whole div